arangod/Replication/DatabaseInitialSyncer.cpp

```diff
@@ -1415,10 +1415,28 @@ Result DatabaseInitialSyncer::handleCollectionsAndViews(VPackSlice const& collSl
     collections.emplace_back(parameters, indexes);
   }
 
-  // STEP 1: validate collection declarations from master
+  // STEP 1: now that the collections exist create the views
+  // this should be faster than re-indexing afterwards
+  // ----------------------------------------------------------------------------------
+
+  if (!_config.applier._skipCreateDrop &&
+      _config.applier._restrictCollections.empty() &&
+      viewSlices.isArray()) {
+    // views are optional, and 3.3 and before will not send any view data
```

Comment:
Wondering about this comment... In 3.3 we didn't even have views. They were introduced with 3.4.
So what should this comment express? Is there any behavior change for views in any view-enabled version?